native/cocos/gi/light-probe/Delaunay.h

```diff
@@ -59,6 +59,13 @@ struct Edge {
     : tetrahedron(tet), index(i), vertex0(v0), vertex1(v1) {
     }
 
+    inline void set(int32_t tet, int32_t i, int32_t v0, int32_t v1) {
```

Comment:
Since c++17, `inline` means "multiple definitions are permitted" rather than "inlining is preferred".
Here `inline` can be omitted in class definition.
https://en.cppreference.com/w/cpp/language/inline